src/pages/inventory/[[...section]].tsx

```diff
@@ -516,6 +545,9 @@ const Drawer = ({
                               }
                             })}
                           </div>
+                          {floorDifference > 10 && (
+                            <p className="flex justify-center text-red-700 font-bold">{`WARNING: ${floorDifference}% below floorpice`}</p>
```

Comment:
```suggestion
                            <p className="flex justify-center text-red-700 font-bold">{`WARNING: ${floorDifference}% below floor price`}</p>
```
Typo here, and added a space between the words.